src/pages/access_review/AppSummary.js

```diff
@@ -74,7 +64,7 @@ export const AppSummary = hh(class AppSummary extends React.Component {
   render() {
     const { darInfo, accessElection, consent, researcherProfile } = this.props;
     const { piName, profileName, institution, department, city, country } = researcherProfile;
-    const mrDAR = JSON.stringify(accessElection.useRestriction, null, 2);
+    const mrDAR = !isNil(accessElection) ? JSON.stringify(accessElection.useRestriction, null, 2) : null;
```

Comment:
address error: DARs with unreviewed or closed election statuses return a 404 when findElectionById is called, in AppSummary the accessElection is only used for the  mrDAR download link, so jus remove that if it is null